quarkus-workshop-super-heroes/docs/src/docs/asciidoc/5a-contract-testing/states.adoc

```diff
@@ -0,0 +1,55 @@
+== States
+
+One of the advantages of Pact over simpler contract verification methods, such as syntax-checking, is that a pact can include a range of expected responses, depending on what was passed in, and external state.
```

Comment:
Maybe when mentioning "syntax-checking", you call out OpenAPI so that the user can associate OpenAPI with a simple syntax checker?